safenode/src/domain/dbc_genesis.rs

```diff
@@ -17,6 +17,12 @@ use thiserror::Error;
 /// Total supply of tokens that will eventually exist in the network: 4,294,967,295 * 10^9 = 4,294,967,295,000,000,000.
 const TOTAL_SUPPLY: u64 = u32::MAX as u64 * u64::pow(10, 9);
 
+/// The secret key for the genesis DBC.
+///
+/// This key is public for auditing purposes. Hard coding its value means all nodes will be able to
+/// validate it.
+pub const GENESIS_DBC_SK: &str = "0c5152498fc5b2f9ed691ef875f2c16f1f950910391f7ba1df63e9f0ce4b2780";
```

Comment:
to mute the github scan warning, consider store this as a file under `resources` dir?

Reply:
Could be better, yes. Practical difference is none, and I would prefer it to be in the actual code. But the warning is annoying..

Reply:
Have asked in slack what the preferred way to handle this is.